File: services/stonks/scripts/sell_script.php
<?php

$wallet_id = $_POST["wallet_id"];
$amount = $_POST["amount"];
$price = (int)htmlentities(file_get_contents("../actives/exchange_cource.exdata"));
$wallet_id_enc = base64_encode($wallet_id);
$money = (int)htmlentities(file_get_contents("../../intercoin/wallets/{$wallet_id_enc}.walletdata"));
$total_price = $price * $amount;
$old_amount = htmlentities(file_get_contents("../actives/{$wallet_id_enc}.transaction"));

if($amount > $old_amount){echo "<script>location.href=\"../../intercoin/error.html?from=../stonks/htmls/sell.html\"</script>";}

else{
shell_exec("py sell_script.py {$wallet_id_enc} {$amount}");
header("Location: ../index.php");
}

?>

File: services/stonks/scripts/sell_script.py
import sys
wallet_id = sys.argv[1]
amount = sys.argv[2]
with open("../actives/exchange_cource.exdata", "r") as exchange_cource_file:
    excource = int(exchange_cource_file.read())
    exchange_cource_file.close()
totalmoney = excource * int(amount)
with open("../../intercoin/wallets/"+wallet_id+".walletdata", "r") as account:    
    balance = account.read()
    new_balance = int(balance) + totalmoney
    account.close()
with open("../../intercoin/wallets/"+wallet_id+".walletdata", "w") as account:
    account.write(str(new_balance))
    account.close()
with open("../actives/"+wallet_id+".transaction", "r") as transaction:
    old_amount = int(transaction.read())
    transaction.close()
with open("../actives/"+wallet_id+".transaction", "w") as transaction:
    transaction.write(str(old_amount - int(amount)))
    transaction.close()